libs/cardano-ledger-api/cardano-ledger-api.cabal

```diff
@@ -55,11 +55,11 @@ library
         aeson >=2.2,
         array,
         cardano-ledger-allegra >=1.1,
-        cardano-ledger-alonzo >=1.4,
+        cardano-ledger-alonzo >=1.5.1,
```

Comment:
And this too should cause incrementing the version of the package to 1.7.0.1, no?

Reply:
Thank you for catching this. I though I got them all.